regolith/tools.py

```diff
@@ -116,6 +117,28 @@ def month_and_year(m=None, y=None):
     return "{0} {1}".format(SHORT_MONTH_NAMES[m], y)
 
 
+def is_since(y, sy, m=1, d=1, sm=1, sd=1):
```

Comment:
Can you please add docstrings for these functions and explain what the meaning of the function arguments is here? Thanks!